Documentation/helm-operator.md

```diff
@@ -58,7 +58,7 @@ To install the helm chart from master, you will need to pass the specific versio
 
 ```console
 helm repo add rook-master https://charts.rook.io/master
-helm search rook-ceph
+helm search repo rook-ceph
```

Comment:
Is the `repo` argument required as of a certain helm version? If it changed recently, we should add a comment to indicate what version requires it.